plugins/search/providers/elastic_search/lib/model/search/kCategorySearch.php

```diff
@@ -26,11 +26,14 @@ public function doSearch(ESearchOperator $eSearchOperator, kPager $pager = null,
 
     protected function initQuery(array $statuses, $objectId, kPager $pager = null, ESearchOrderBy $order = null, ESearchAggregations $aggregations=null)
     {
+        $indexName = kBaseESearch::getElasticIndexNamePerPartner( ElasticIndexMap::ELASTIC_CATEGORY_INDEX, kCurrentContext::getCurrentPartnerId());
         $this->query = array(
-            'index' => ElasticIndexMap::ELASTIC_CATEGORY_INDEX,
+            'index' =>  $indexName,
             'type' => ElasticIndexMap::ELASTIC_CATEGORY_TYPE
         );
 
+        KalturaLog::debug("Index -" . $indexName);
```

Comment:
That debug is a little redundant since we print that info in the getElasticIndexNamePerPartner method
KalturaLog::debug("Using index name $indexName for $partnerId");